test-coverage.md

```diff
@@ -13,11 +13,7 @@ Typically, the essential configuration for gathering code coverage is already pr
 
 ```xml
     ...
-    <coverage>
-        <include>
-            <directory suffix=".php">./app</directory>
-        </include>
-    </coverage>
+    <coverage/>
```

Comment:
I think it makes more sense to update this to the new [`<source>` element](https://docs.phpunit.de/en/10.3/configuration.html#the-source-element) format. 🤷🏻

Reply:
@owenvoke can you pull request it?